### DISCUSSION TITLE
Unable to install packages via pip

### DISCUSSION THREAD

**Discussion by 0ShNa0** [community/community#57434] [Actions]
### Select Topic Area

General

### Body

trying to install python packages and it always shows NewConnectionError
when i enter pip install <package_name> 

OS:MacOS

**Comment by backtoaldo** [comment]
Could you provide the error image or text that appears right after you type the command?
Could you also comment on which version of MacOS you are using?

It is very likely that it is an error due to the Proxy. If you use VPN, it may have changed your network to IPV6, maybe by turning it off you will be able to install it. 

If you can prove more information, I can try to help.

**Reply by 0ShNa0** [reply]
![WhatsApp Image 2023-06-07 at 21 18 13](https://github.com/community/community/assets/72717361/3fed2e62-8e0a-4ad7-83f4-5885740ab35a)


MacOS Monterey v12.6
Thankyou for helping out!

**Reply by backtoaldo** [reply]
Ok, [Errno 8] nodename nor servname provided, or not know.  

Could you tell me if at the end of this error message, there are these two lines?

```
ERROR: Could not find a version that satisfies the requirement requests-X.XX.X (from versions: none)
ERROR: No matching distribution found for requests-X.XX.X
```

If there is this message, you can try installing with a different command, for some reason the syntax is different:

`pip install requests==X.XX.X`

In place of the X.XX.X, put the version that appears in your terminal.

**Reply by 0ShNa0** [reply]
Showed me the same error you mentioned in the last comment(ERROR messages)

**Reply by backtoaldo** [reply]
Try this:

```
python3 -m pip download \
  --dest=my-download-directory \
  --no-binary=:all: \
  requests==X.XX.X
```

**Reply by 0ShNa0** [reply]
Same error as initally posted

**Comment by airtower-luna** [comment]
Which package are you trying to install? The message in the screen photo is incomplete, but kinda looks like there's an invalid URL. :thinking:

**Reply by 0ShNa0** [reply]
Every package seems to give the same error.But I need to install semantic kernel asap
